DustRiders/DustRiders/Overlay.cpp

```diff
@@ -374,6 +374,26 @@ void Overlay::RenderPause(StateHandler::GameState prevState, int windowHeight, i
 	ImGui::Text("ESC or START to toggle pause");
 	ImGui::End();
 
+	ImGui::SetNextWindowPos(ImVec2(windowWidth * 1.6, windowHeight * 0.6));
```

Comment:
@brooke-k can you confirm this looks good on your 1080p screen

Reply:
Pushed a commit moving the text up slightly so it isn't cut off at the bottom of the screen (a one character change)